### DISCUSSION TITLE
Rename `q` to better represent its meaning

### DISCUSSION THREAD

**Discussion by chris-ashe** [ukaea/PROCESS#3106] [Ideas]
This is a follow on from #321 which did a quick variable rename to make the definition of `q` clearer in more places.

The definition of `q` does not initially give the impression that it is the safety factor near the edge of the plasma but is instead a function of plasma radius.
Ideas about renaming `q` to a different variable `qedge`  etc are possible though will make old input files temporarily broken until they are all updated.  This definition also changes meaning depending on what model are being ran; stellarator/plasma current scalings etc.

**Comment by grmtrkngtn** [comment]
So there shouldn't be one "q" variable, and have unique ones for each definition?

**Comment by grmtrkngtn** [comment]
Can we close this discussion now @chris-ashe ?

**Comment by chris-ashe** [comment]
Closed with #3108
